packages/stream_video/lib/src/call/state/mixins/state_coordinator_mixin.dart

```diff
@@ -414,8 +414,8 @@ mixin StateCoordinatorMixin on StateNotifier<CallState> {
           audioLevel: e.audioLevel,
           isSpeaking: e.isSpeaking,
           isDominantSpeaker: e.isDominantSpeaker,
-          isPinned: e.isPinned,
           viewportVisibility: e.viewportVisibility,
+          pin: e.pin,
```

Comment:
I think this makes it a breaking change right? Or are these mixins not part of the public API or the core package?